trillian/ctfe/requestlog.go

```diff
@@ -125,6 +131,11 @@ func (dlr *DefaultRequestLog) LeafHash(_ context.Context, lh []byte) {
 	glog.V(vLevel).Infof("RL: LeafHash: %s", hex.EncodeToString(lh))
 }
 
+// SCT logs an SCT that will be issued to a client.
+func (dlr *DefaultRequestLog) SCT(_ context.Context, sct []byte) {
+	glog.V(vLevel).Infof("RL: Issuing SCT: %s", hex.EncodeToString(sct))
```

Comment:
Wouldn't a format specifier of %x do the trick without `hex.EncodeToString` ?

Reply:
Done.